ivy/functional/backends/jax/activations.py

```diff
@@ -7,33 +7,67 @@
 import jax.numpy as jnp
 
 # local
+import ivy
 from ivy.functional.backends.jax import JaxArray
 
 
-def relu(x: JaxArray) -> JaxArray:
-    return jnp.maximum(x, 0)
+def relu(x: JaxArray, out: Optional[JaxArray] = None) -> JaxArray:
+    ret = jnp.maximum(x, 0)
+    if ivy.exists(out):
+        return ivy.inplace_update(out, ret)
+    return ret
 
 
-def leaky_relu(x: JaxArray, alpha: Optional[float] = 0.2) -> JaxArray:
-    return jnp.where(x > 0, x, x * alpha)
+def leaky_relu(
+        x: JaxArray,
+        alpha: Optional[float] = 0.2,
+        out: Optional[JaxArray] = None
+) -> JaxArray:
+    ret = jnp.where(x > 0, x, x * alpha)
+    if ivy.exists(out):
```

Comment:
we should not handle `out` in the backend. We should remove the `out` argument here unless the backend function itself is able to handle this for us.